src/fsharp/FSharp.Core/math/z.fsi

```diff
@@ -1,82 +1,7 @@
-        /// Convert a big integer to a floating point number
-        static member op_Explicit : x:BigInteger -> float
-        /// Convert a big integer to a 64-bit signed integer
-        static member op_Explicit : x:BigInteger -> int64
-        /// Convert a big integer to a 32-bit signed integer
-        static member op_Explicit : x:BigInteger -> int32
-        /// Parse a big integer from a string format
-        static member Parse    : text:string -> BigInteger
-        /// Return the sign of a big integer: 0, +1 or -1
-        member Sign    : int
-        /// Compute the ratio and remainder of two big integers
-        static member DivRem : x:BigInteger * y:BigInteger * [<System.Runtime.InteropServices.Out>]rem:BigInteger byref -> BigInteger
-
-        /// This operator is for consistency when this type be used from other CLI languages
-        static member op_LessThan           : x:BigInteger * y:BigInteger -> bool
-        /// This operator is for consistency when this type be used from other CLI languages
-        static member op_LessThanOrEqual    : x:BigInteger * y:BigInteger -> bool
-        /// This operator is for consistency when this type be used from other CLI languages
-        static member op_GreaterThan        : x:BigInteger * y:BigInteger -> bool
-        /// This operator is for consistency when this type be used from other CLI languages
-        static member op_GreaterThanOrEqual : x:BigInteger * y:BigInteger -> bool
-        /// This operator is for consistency when this type be used from other CLI languages
-        static member op_Equality             : x:BigInteger * y:BigInteger -> bool
-        /// This operator is for consistency when this type be used from other CLI languages
-        static member op_Inequality           : x:BigInteger * y:BigInteger -> bool
-
-        /// Return the greatest common divisor of two big integers
-        static member GreatestCommonDivisor : x:BigInteger * y:BigInteger -> BigInteger
-        /// Return n^m for two big integers
-        static member Pow    : x:BigInteger * y:int32 -> BigInteger
-        /// Compute the absolute value of a big integer 
-        static member Abs    : x:BigInteger -> BigInteger
-        /// Get the big integer for zero
-        static member Zero    : BigInteger 
-        /// Get the big integer for one
-        static member One     : BigInteger 
-
-        /// Return true if a big integer is 'zero'
-        member IsZero : bool
-        /// Return true if a big integer is 'one'
-        member IsOne : bool
-        interface System.IComparable
-        override Equals : obj -> bool
-        override GetHashCode : unit -> int
-        override ToString : unit -> string
-
-        /// Construct a BigInteger value for the given integer
-        new : x:int -> BigInteger
-        /// Construct a BigInteger value for the given 64-bit integer
-        new : x:int64 -> BigInteger
-#endif
-
+namespace Microsoft.FSharp.Math
```

Comment:
This should provide a detailed description of why we are leaving this namespace in place even though there are no types  in this.